ci-operator/step-registry/enable-qe-catalogsource/enable-qe-catalogsource-commands.sh

```diff
@@ -122,8 +122,9 @@ EOF
 function create_catalog_sources()
 {
     # get cluster Major.Minor version
-    ocp_version=$(oc version -o json | jq -r '.openshiftVersion' | cut -d '.' -f1,2)
-    index_image="quay.io/openshift-qe-optional-operators/aosqe-index:v${ocp_version}"
+    kube_major=$(oc version -o json |jq -r '.serverVersion.major')
```

Comment:
I remember, for low OCP 4.x, such as, 4.5, 4.6. There is no `serverVersion.major` field, I will confirm it later.

Reply:
Anyway, we can use it first.

Reply:
OK